packages/frontend/src/mantine8Theme.ts

```diff
@@ -153,6 +154,22 @@ export const getMantine8ThemeOverride = (
                     loaders: { ...Loader.defaultLoaders, dots: DotsLoader },
                 },
             }),
+
+            Select: Select.extend({
+                defaultProps: {
+                    radius: 'md',
+                },
+                styles: (theme) => ({
+                    input: {
+                        borderColor: theme.colors.gray[2],
+                        boxShadow: `var(--mantine-shadow-subtle)`,
+                        color: theme.colors.gray[7],
+                    },
+                    dropdown: {
+                        paddingRight: 4,
+                    },
+                }),
```

Comment:
Do we want these styles to be applied to all `Select`s under this mantine provider? I don't think so 🤔

Reply:
good point. I opted for this (instead of variants) because I like the idea of consistency and having all selects with the same style by default. We can always create specialized ones by updating the styles in the component itself.
But let me know, I can move these into the switcher instead!

Reply:
I'm just wondering if these would look a lot different if on a form for example - leading to making the changes to those components as well, like `TextInput`, `TagInput` etc
if you could move then the `styles` to the component itself that would be great - and then we can move the theme if we want to have it for all input-like components. wdyt?

Reply:
sounds good! Will move! 🙏